operations/kubernetes/Kyverno.md

```diff
@@ -392,6 +392,41 @@ Example output:
 "cray-ceph-csi-cephfs-nodeplugin","DaemonSet","ceph-cephfs","disallow-privileged-containers","validation error: Privileged mode is disallowed. The fields spec.containers[*].securityContext.privileged and spec.initContainers[*].securityContext.privileged must be unset or set to `false`. Rule autogen-privileged-containers failed at path /spec/template/spec/containers/0/securityContext/privileged/"
 ```
 
+## What is new in the HPE CSM 1.6 release and above
+
+1. Kyverno is upgraded from 1.9.5 version to 1.10.7 version and is now available for customers as part of the HPE CSM 1.6 release.
+
+   This is a major upgrade with many new features and bug fixes. For complete list please refer to the link [CHANGELOG](https://github.com/kyverno/kyverno/blob/main/CHANGELOG.md)
+
+2. Container Image Signature Verification Kyverno Policy.
+
+### Container Image Signature Verification Kyverno Policy
+
+Container image signing and runtime verification policy by name `check-image` is delivered (through Kyverno
+policy engine) as part of CSM 1.6 release in `Audit` only mode. (`Audit` only mode will log the policy violation
+warning messages in to the cluster report and as events)
+
+By default the `check-image` policy is shipped as a `ClusterPolicy` and is configured to work in a sample (non-existent) namespace. To enable the policy, end users must customize it to the targeted namespaces.
+
+CSM runs in an air gapped environment, where the images stored in `artifactory` are mirrored to the local nexus registry. Kyverno policy engine does not support
+this environment. To enable Kyverno policy for such environments, all `image specs` need to be replaced from `artifactory.algol60.net/csm-docker/stable/image:tag`
+to `registry.local/artifactory.algol60.net/csm-docker/stable/image:tag` (prepend `registry.local/` to the image spec). This will force Kyverno to check local nexus
```

Comment:
```suggestion
to `registry.local/artifactory.algol60.net/csm-docker/stable/image:tag` (prepend `registry.local/` to the image spec). This will force Kyverno to check local Nexus
```